security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -13,6 +13,20 @@ These release notes track the development of the File Integrity Operator in the
 
 For an overview of the File Integrity Operator, see xref:../../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[Understanding the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-0-1-32"]
+== OpenShift File Integrity Operator 0.1.32
+
+The following advisory is available for the OpenShift File Integrity Operator 0.1.32:
+
+* link:https://access.redhat.com/errata/RHBA-2022:7095[RHBA-2022:7095 OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-0-1-32-bug-fixes"]
+=== Bug fixes
+
+* Previously, alerts issued by the File Integrity Operator did not set a namespace, making it difficult to understand where the alert originated. Now, the Operator sets the appropriate namespace, increasing understanding of the alert. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2112394[*BZ#2112394*])
```

Comment:
Suggestion: "...making it difficult to understand from which namespace the alert originated."

Reply:
"increasing understanding of the alert" - this is a little awkward, maybe "providing more information about the alert" or similar?

Reply:
@kcarmichael08 Thanks for the suggestion! Updates have been made. :)